### ISSUE TITLE
Some UI note

### ISSUE THREAD

**Issue by uponatime2019** [solidlyexchange/solidly.exchange#12]
### 1. Disable filter "My Deposits" option
<img width="1339" alt="Screen Shot 2022-02-27 at 18 19 05" src="https://user-images.githubusercontent.com/15900671/155880438-017cecb9-6f62-48dd-b0a9-08109a37895a.png">
This option should turn off by default, It take me for weeks to figure it out

### 2. Fix Swap button on Mobile version
![IMG_1563](https://user-images.githubusercontent.com/15900671/155880496-6c41c80a-ebb7-42c6-86dd-c8a9d6fa718e.jpg)
I also note that Balance loading very low

**Comment by antonnell** [comment]
1. Disable filter "My Deposits" option
  That's a matter of opinion. Most people don't need to see 123123 pairs.

2. Fix Swap button on Mobile version
How do you manage to get the swap button that big? Can't reproduce on my mobile or by messing with resolutions on PC.

**Comment by uponatime2019** [comment]
1. When I go to Liquidity, I expect to see something, I though other users will, right? If all empty, I thought nothing to see here, go other tabs to explore this project. Bc it's my habit learned by other dapps
2. I'm using Metamask from latest iOS from iPhone pro max, you can consider

**Comment by antonnell** [comment]
Uniswap behaviour when you go to pools
![image](https://user-images.githubusercontent.com/38530524/155885188-6b1b55b4-3c1b-4359-9c90-9daa0caf982e.png)

Sushiswap behaviour when you go to pools
![image](https://user-images.githubusercontent.com/38530524/155885250-8229bcd8-ddb4-4487-8736-77a97243ef07.png)

I guess if you are referring to "Farm" functionality then I guess Sushi  has a "Your Farms" vs "All Farms" option, much like my filters toggle has.
![image](https://user-images.githubusercontent.com/38530524/155885320-044168aa-1722-4b34-a85a-d03d7473a6b6.png)

I'll consider changing things once the UI becomes more "Farm" focucssed as opposed to liquidity focussed.

**Comment by antonnell** [comment]
I'll add some messaging in the empty table to assis users as well.
"No liquidity found, please change filters and search to discover liquidity pools"
Something like that.

Thanks for the feedback!

**Comment by uponatime2019** [comment]
> I'll add some messaging in the empty table to assis users as well.
> 
> "No liquidity found, please change filters and search to discover liquidity pools"
> 
> Something like that.
> 
> 
> 
> Thanks for the feedback!

Yep PancakeSwap it's my first DEX, it's more UI friendly than Uniswap, and Pools and Farms can compine without maker user confused. Thanks for consider. 
And one more things, if you dont mind web resource cost, may be consider make all info available to users not connect wallet yet, but can view pools, votes and other infos not require a wallet

**Comment by antonnell** [comment]
> And one more things, if you dont mind web resource cost, may be consider make all info available to users not connect wallet yet, but can view pools, votes and other infos not require a wallet

Yea, that's one thing I would still like to add. But it is low priority while getting things up and running.

**Comment by uponatime2019** [comment]
As a software developer I know one thing: Ux matter, very much. As a exprienced user, I can said (in my view, of course), project should consider: 
1. Swap problem, some route have 99% loss
2. Add liquidity option (just add LP or staked, user always stupid, like me), bc Solidly don't dislay APR, which make users consider stake or not. If something can't fix easy, give user an easy understand guide
3. Other UI problem (connect phone metamask, trust, math wallets), imho UI ugly, not much wow as Alpha Finance
Thank you, from a Fantom lover

**Comment by ches** [comment]
> 1. Disable filter "My Deposits" option
>    That's a matter of opinion. Most people don't need to see 123123 pairs.

Just a piece of anecdata: I replied to literally dozens of people across a few Discord servers to point out that they needed to untick this filter, people kept coming along who didn't understand how to add liquidity or thought Solidly was broken on first impression because they didn't see any pools. It quickly needed to be a pinned post, with a screenshot circling the dropdown.

So I agree with @uponatime2019 that this needs _some_ kind of improved UX solution, if changing the default state of the filter is not one you prefer @antonnell.
